src/components/content/heroes/aurora/aurora-components.ts

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from 'styled-components';
+
+const BACKGROUND_LAYER_Z = -2;
+const FOREGROUND_LAYER_Z = -1;
+
+interface AuroraBackgroundProps {
+  source: string;
+  overwriteBackground?: string;
+}
+
+export const AuroraBackground = styled.div<AuroraBackgroundProps>`
+  ${(props) =>
+    props.overwriteBackground !== undefined
```

Comment:
awesome, thanks for accepting and implementing my proposal regarding removing dependencies